src/OrbitGl/CaptureWindow.cpp

```diff
@@ -392,7 +405,7 @@ std::unique_ptr<AccessibleInterface> CaptureWindow::CreateAccessibleInterface()
 void CaptureWindow::Draw() {
   ORBIT_SCOPE("CaptureWindow::Draw");
   uint64_t start_time_ns = orbit_base::CaptureTimestampNs();
-  bool time_graph_was_redrawn = false;
+  bool time_graph_was_redrawn = time_graph_ != nullptr ? time_graph_->IsRedrawNeeded() : false;
```

Comment:
2 things:
a)
I think that 
`bool time_graph_was_redrawn = time_graph_ != nullptr && time_graph_->IsRedrawNeeded() `

is easier to read

b) Shouldn't be `!time_graph_->IsRedrawNeeded`?

Reply:
a) True
b) No, this determines if timegraph had `RedrawNeeded` set before it is cleared lateron.